auv_control_demos/README.md

```diff
@@ -0,0 +1,321 @@
+# AUV Control Demos
+
+This package contains examples that demonstrate how to use the controllers in
+`auv_controllers`. There are two examples. The first shows how to launch just a
+single controller on its own. The second example shows how to chain the
+controllers together to a mock hardware interface for an AUV.
+
+## Example 1: Individual Controller
+
+This example shows how to launch an individual auv controller connect them to a
+mock hardware interface. We launch the integral sliding mode controller which
+takes velocity reference and state interfaces and outputs a wrench command
+interface.
+
+The launch file [individual.launch.py](https://github.com/Robotic-Decision-Making-Lab/auv_controllers/blob/main/auv_control_demos/launch/individual.launch.py)
+contains the code to launch the integral sliding mode controller to a mock
+hardware interface, and that is the launch file we use for this first example.
+
+### Tutorial Steps
+
+1. To test the chainable controller framework, run the following command:
+
+    ```bash
+    ros2 launch auv_control_demos individual.launch.py
+    ```
+
+    The launch file loads and starts the mock hardware and the controller.
```

Comment:
Reword to "The launch file loads and starts the hardware interface and controller."

Reply:
done